core/src/Focusable.ts

```diff
@@ -1044,8 +1046,6 @@ export class FocusableAPI implements Types.FocusableAPI {
                 return false;
             }
 
-            attrVal = e.getAttribute('aria-disabled');
```

Comment:
Something is fishy here. I suppose you were willing to check the new prop, but have just removed the line unconditionally.